cmd/gds/main.go

```diff
@@ -874,9 +874,9 @@ func adminListVASPs(c *cli.Context) (err error) {
 	defer cancel()
 
 	params := &admin.ListVASPsParams{
-		Page:     c.Int("page"),
-		PageSize: c.Int("page-size"),
-		Status:   c.String("status"),
+		Page:          c.Int("page"),
+		PageSize:      c.Int("page-size"),
+		StatusFilters: c.StringSlice("status-filters"),
```

Comment:
Nice! Don't forget you also have to change the flag definition on [line 367](https://github.com/trisacrypto/directory/blob/main/cmd/gds/main.go#L367)